### ISSUE TITLE
[Bug]: Clean install doesn't work as described

### ISSUE THREAD

**Issue by maniac-0s** [Panchovix/stable-diffusion-webui-reForge#217]
### Checklist

- [ ] The issue exists after disabling all extensions
- [ ] The issue exists on a clean installation of webui
- [ ] The issue is caused by an extension, but I believe it is caused by a bug in the webui
- [ ] The issue exists in the current version of the webui
- [ ] The issue has not been reported before recently
- [ ] The issue has been reported before but has not been fixed yet

### What happened?

The README says "Then run webui-user.bat (Windows) or webui-user.sh (Linux)." but that does nothing because the file contains nothing except variables that are all commented out.

### Steps to reproduce the problem

Follow the Readme

### What should have happened?

Change that to "run webui.bat or webui.sh"

### Sysinfo

-

### Console logs

```Shell
-
```

### Additional information

-

**Comment by Panchovix** [comment]
Hi there, those files should let you open the UI. Basically of them call the webui bat/sh file and you can add some parameters.

The only commented variables are related to re-use A1111 paths.

**Comment by maniac-0s** [comment]
No, the [file](https://github.com/Panchovix/stable-diffusion-webui-reForge/blob/main/webui-user.sh) is empty except for comments. Nothing happens when you run it.

**Comment by Panchovix** [comment]
Okay I see it now, for the linux version you have to modify the file according to your needs and point to python, etc

I have updated the README.
